gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/source/extractor/extract/kafka/KafkaExtractorStatsTracker.java

```diff
@@ -306,6 +310,13 @@ public void onDecodeableRecord(int partitionIdx, long readStartTime, long decode
         if (logAppendTimestamp > 0 && (System.currentTimeMillis() - logAppendTimestamp > recordLevelSlaMillis)) {
           v.slaMissedRecordCount++;
         }
+        // See context in GOBBLIN-2114
```

Comment:
Can you add the short description here instead of referencing the Gobblin ticket? Since we have so many stats tracker here, it would be good to have a short description so we know what we are tracking and going forward, we know what we already have.
The naming for the variables might be more descriptive/understandable maybe as appendDurationTime and creationDurationTime to describe that it is amount of time taken between HDFS and when it was produced / appended. What do you think?

Reply:
comment addressed